components/public-api/gitpod/v1/scm.proto

```diff
@@ -3,9 +3,70 @@ syntax = "proto3";
 package gitpod.v1;
 
 import "google/protobuf/timestamp.proto";
+import "gitpod/v1/pagination.proto";
 
 option go_package = "github.com/gitpod-io/gitpod/components/public-api/go/v1";
 
+service SCMService {
```

Comment:
@AlexTugarev i have a feeling it is alraedy on main, do you need a PR rebase?

Reply:
So much got used to stacked PRs. Now, with squash on merge, this is causes extra loops and feels broken.